options.go

```diff
@@ -76,6 +76,10 @@ type Options struct {
 	// Default is ReadTimeout.
 	WriteTimeout time.Duration
 
+	// Type of Pool
+	// Default is lifo, supports fifo too
+	// Note that fifo has higher overhead compared to lifo
+	PoolType pool.PoolType
```

Comment:
`pool` is a private package. Perhaps `PoolFIFO bool`?

Reply:
Thanks for the headsup, I'll do that .

Reply:
I've changed it, along with the golint issues.